profile/README.md

```diff
@@ -42,11 +42,11 @@ documentation
 
 * [Dart - at_client_sdk](https://github.com/atsign-foundation/at_client_sdk)
 * [Java - at_java](https://github.com/atsign-foundation/at_java)
-* [MicroPython - at_pico_w](https://github.com/atsign-foundation/at_pico_w)
-* [C++ - at_esp32](https://github.com/atsign-foundation/at_esp32)
-* [C - at_c](https://github.com/atsign-foundation/at_c)
-* [Rust - at_rust](https://github.com/atsign-foundation/at_rust)
-* [Python - at_python](https://github.com/atsign-foundation/at_python)
+* [MicroPython - at_pico_w (beta)](https://github.com/atsign-foundation/at_pico_w)
```

Comment:
Rather than having (beta) on each line can we have a Beta SDKs subsection (and maybe some text explaining that they're still works in progress).

Reply:
<img width="1011" alt="image" src="https://github.com/atsign-foundation/.github/assets/79019866/401b689b-47b7-42c1-918f-0cc02464291f">

https://github.com/atsign-foundation/.github/pull/60/commits/8b12a11381d8b8aade23b24b4bcc4a6199f8daaa

Do you think a header 3 is good?